### ISSUE TITLE
Some text autohyphenated on word wrap with  pango 1.44.7

### ISSUE THREAD

**Issue by lukefromdc** [mate-desktop/pluma#520]
#### Expected behaviour
Word wrap never adds anything to text, even when a string of characters with no whitespace must be split to enable wrap, and no matter what the line of characters contains

#### Actual behaviour
If a line of text does not contain enough whitespace to respect "do not split words across two lines" and is split at a </a> sequence a hyphen is inserted at the end of that line of text. This can seriously confusing coding, a common use case for pluma. If you disable "do not split words across two lines" you will now see autohyphenation at the end of ANY wrapped line of test

Workaround is to resize the window large enough to display the entire line of text,  or disable word wrap.

This is more fallout from Pango's new auto-hyphenation "feature" which is enabled by default and has to be explicitly turned off:
https://gitlab.gnome.org/GNOME/nautilus/merge_requests/482/diffs

We also had this in caja, fixed by 
https://github.com/mate-desktop/caja/issues/1284
and
https://github.com/mate-desktop/caja/pull/1334

#### Steps to reproduce the behaviour
Open Pluma, create a long line of text with no whitespace and including the html </a> sequence, enable word wrap, resize the window to put the word wrap aplit during that sequence. Probably other character strings are also effected.

Now turn off  "do not split words across two lines" but leave word wrapping enabled,  open a new Pluma tab, and just hold down any normal letter key to produce an extended line of characters. When it goes to a second line, a hyphen appears at the end of the first line of text.

#### MATE general version
1.23.1

#### Package version
Locally compiled Pluma 1.23.3

#### Linux Distribution
Debian Unstable with libpango 1.44.7-1 installed from Debian Experimental. Note that Debian has NOT moved this to Unstable, no doubt because of that auto-hyphenation issue. Thus Debian users will NOT normally see this as of today

#### Link to downstream report of your Distribution
None as this is a locally compiled package

**Comment by N0rbert** [comment]
Downstream bug-report for Pluma (and actually Pango) on LaunchPad is https://bugs.launchpad.net/bugs/1909806 .

**Comment by lukefromdc** [comment]
I agree that autohyhenation is a bug not a feature anywhere I've ever seen it. Ideally everytime we use pango to render more than one line, we should explicity turn off autohypenation.
